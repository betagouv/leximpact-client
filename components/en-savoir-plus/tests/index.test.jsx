/* eslint
    indent: [2, 2],
    semi: [2, "always"],
    react/jsx-indent: [2, 2],
    react/jsx-indent-props: [2, 2]
    max-nested-callbacks: [2, { "max": 4 }]
*/
import { createShallow } from "@material-ui/core/test-utils";

import MentionsLegales from "../index";

describe("components | app-header | index", () => {
  let shallow;
  beforeAll(() => {
    const options = { dive: true };
    shallow = createShallow(options);
  });

  describe("snapshot", () => {
    it("doit correspondre quand l'user n'est pas connectÃ©", () => {
      const props = {};
      const wrapper = shallow(<MentionsLegales {...props} />);
      expect(wrapper).toBeDefined();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
