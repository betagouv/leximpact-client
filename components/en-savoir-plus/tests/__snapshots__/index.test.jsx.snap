// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | app-header | index snapshot doit correspondre quand l'user n'est pas connecté 1`] = `
<div
  className="MentionsLegales-container-2"
>
  <WithStyles(Grid)
    alignItems="flex-start"
    container={true}
    direction="row"
    justify="space-between"
  >
    <WithStyles(Grid)
      className="MentionsLegales-gridItemText-5"
      item={true}
      xs={6}
    >
      <WithStyles(Typography)>
        <span
          className="MentionsLegales-spanTitreIntro-7"
        >
          OPEN LexImpact, c'est quoi ?
        </span>
      </WithStyles(Typography)>
      <WithStyles(Typography)>
        OPEN LexImpact est une interface qui permet de simuler, de façon rapide, l'impact des réformes paramétriques de l'impôt sur le revenu sur des foyers fiscaux types.
      </WithStyles(Typography)>
      <a
        className="MentionsLegales-alink-1"
        href="/presentation-et-cgu"
      >
        en savoir plus
      </a>
    </WithStyles(Grid)>
    <WithStyles(Grid)
      className="MentionsLegales-gridItemButtons-4"
      item={true}
      xs={6}
    >
      <div>
        <div
          className="MentionsLegales-divButton-3"
        >
          <Link
            href="/presentation-et-cgu"
          >
            <WithStyles(Button)
              color="inherit"
              fullWidth="true"
              size="large"
              variant="outlined"
            >
              Présentation et conditions d'utilisation
            </WithStyles(Button)>
          </Link>
        </div>
        <div
          className="MentionsLegales-divButton-3"
        >
          <Link
            href="/mentions-legales"
          >
            <WithStyles(Button)
              color="inherit"
              fullWidth="true"
              size="large"
              variant="outlined"
            >
              Mentions légales
            </WithStyles(Button)>
          </Link>
        </div>
        <div
          className="MentionsLegales-divButton-3"
        >
          <WithStyles(Button)
            color="inherit"
            fullWidth="true"
            href="mailto:contact@leximpact.beta.gouv.fr"
            size="large"
            variant="outlined"
          >
            Une question ? Un bug ? Un avis ?
          </WithStyles(Button)>
          <p
            className="MentionsLegales-pAdresseMail-6"
          >
            Écrivez-nous à contact@leximpact.beta.gouv.fr !
          </p>
        </div>
      </div>
    </WithStyles(Grid)>
  </WithStyles(Grid)>
</div>
`;
